core/src/main/java/org/apache/calcite/sql/fun/SqlBitOpAggFunction.java

```diff
@@ -39,7 +39,7 @@ public class SqlBitOpAggFunction extends SqlAggFunction {
 
   //~ Constructors -----------------------------------------------------------
 
-  /** Creates a SqlBitOpAggFunction. */
+  /** Creates a SqlBitOpAggFunction from a SqlKind. */
```

Comment:
nit: What do you think about changing the previous call sites of this to call the new function with 
`SqlBitOpAggFunction(call.getKind().getName(), call.getKind())`?

Reply:
I'm indifferent, the standard operators can derive the name from the `SqlKind`, the library-specific ones cannot so they need the name `specified`. Whether it's refactored so they can use the same constructor doesn't matter too much to me

Reply:
Actually re-reading this I think I would prefer keeping the `SqlBitOpAggFunction(SqlKind kind)` constructor, I think it makes sense to only pass in one parameter if possible.